karma.conf.js

```diff
@@ -13,8 +13,7 @@ if (process.env.TRAVIS) {
     'SL_EDGE',
     'SL_CHROME',
     'SL_FIREFOX',
-    'SL_ANDROID',
-    'SL_SAFARI'
```

Comment:
y u no safari?

Reply:
If you look in the master, the CI is failing.

Reply:
y u no fix?

Reply:
I've locked down the MacOS version. This means when newer Safari versions will appear, we will still test on Safari 12.